### PR TITLE
feat: Message input - theme-v2 #93

### PR DESCRIPTION
**Pull request by szuperaz**
Details: https://github.com/GetStream/stream-chat-angular/pull/297

### CODE REVIEW

File: src/v2/styles/_theme-variables.scss [hunk 1]
```diff
@@ -1,3 +1,9 @@
+// Since Angular uses dynamic components it's not enough the hide the message input for theme-v1/v2, we have to remove it from the DOM tree
+// Angular component lifecycle requires us to place this on the root level
+:root {
```
Comment (szuperaz) [inline src/v2/styles/_theme-variables.scss:3]: Not happy to add a root level variable but couldn't find a better solution (reason described above in the source code)

File: src/v2/styles/Channel/Channel-layout.scss [hunk 2]
```diff
@@ -15,6 +15,7 @@
       display: flex;
       flex-direction: column;
       width: 100%;
+      min-width: 0;
```
Comment (petyosi) [inline src/v2/styles/Channel/Channel-layout.scss:18]: Why's that necessary?

**Review by petyosi — APPROVED** [review]

File: src/v2/styles/Channel/Channel-layout.scss (hunk 2 above)
Comment (petyosi) [reply src/v2/styles/Channel/Channel-layout.scss:18]: Thanks, did not know about this!

**Pull request merged**